clang/lib/DPCT/RulesLang/RulesLang.cpp

```diff
@@ -3567,9 +3565,24 @@ void EventAPICallRule::handleEventElapsedTime(bool IsAssigned) {
         Assginee = getStmtSpelling(UO->getSubExpr());
     }
 
-    auto StartTimeStr = StmtStrArg1 + "->get_profiling_info<"
+    const auto *StartArg = TimeElapsedCE->getArg(1)->IgnoreImpCasts();
+    const auto *EndArg = TimeElapsedCE->getArg(2)->IgnoreImpCasts();
+
+    ExprAnalysis EAForStartArg(StartArg);
+    ExprAnalysis EAForStopArg(EndArg);
```

Comment:
How about we align to use the End?
```suggestion
    ExprAnalysis EAForEndArg(EndArg);
```